dids/src/test/kotlin/web5/sdk/dids/methods/jwk/DidJwkTest.kt

```diff
@@ -114,4 +118,19 @@ class DidJwkTest {
       assertEquals(JsonCanonicalizer(expectedJson).encodedString, JsonCanonicalizer(didDocument.toJson()).encodedString)
     }
   }
+}
+
+class Web5TestVectorsDidJwkTest {
```

Comment:
we took away the Test part

it should be Web5TestVectorsDidJwk

Reply:
Excellent catch! Thank you and fixed!